packages/components/src/toggle-group-control/test/index.js

```diff
@@ -38,4 +53,36 @@ describe( 'ToggleGroupControl', () => {
 		fireEvent.click( firstRadio );
 		expect( mockOnChange ).toHaveBeenCalledWith( 'rigas' );
 	} );
+	it( 'should render tooltip where `showTooltip` === `true`', async () => {
+		render(
+			<ToggleGroupControl label="Test Toggle Group Control">
+				{ optionsWithTooltip }
+			</ToggleGroupControl>
+		);
+		const firstRadio = screen.getByLabelText(
+			'Click for Delicious Gnocchi'
```

Comment:
Thank you for adding these unit tests!

(also, I wish the "click for gnocchi" was real!)

Reply:
My Nonna is on speed dial. Same thing. 😆